<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>695</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>751</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover{
color:black;
border-top    : 1px solid rgba(200, 200, 200, 255);
border-left   : 1px solid rgba(200, 200, 200, 255);
border-right  : 2px solid rgba(117, 117, 117, 255);
border-bottom : 2px solid rgba(117, 117, 117, 255);}
</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QPushButton" name="pushButton_max">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(69,173,246);
border-radius:6px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_cv">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>391</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:gray;
border-top          : 1px solid rgba(200, 200, 200, 255);
 border-left         : 1px solid rgba(200, 200, 200, 255);
    border-right        : 2px solid rgba(117, 117, 117, 255);
    border-bottom       : 2px solid rgba(117, 117, 117, 255);</string>
    </property>
    <property name="text">
     <string>视频显示区</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mini">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(151,212,75);
border-radius:6px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_shutdown">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255,72,77);
border-radius:6px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>711</width>
      <height>271</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QListView" name="listView_result">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>681</width>
       <height>241</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>681</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">    border-top          : 1px solid rgba(200, 200, 200, 255);
    border-left         : 1px solid rgba(200, 200, 200, 255);
    border-right        : 2px solid rgba(117, 117, 117, 255);
    border-bottom       : 2px solid rgba(117, 117, 117, 255);
background-color:rgb(160,160,160);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>371</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>This is result ......</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clearList">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>0</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0,162,232);
border-radius:4px;
color:white;</string>
     </property>
     <property name="text">
      <string>Clear List</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_chosenImg">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>241</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:gray;
border-top          : 1px solid rgba(200, 200, 200, 255);
    border-left         : 1px solid rgba(200, 200, 200, 255);
    border-right        : 2px solid rgba(117, 117, 117, 255);
    border-bottom       : 2px solid rgba(117, 117, 117, 255);</string>
    </property>
    <property name="text">
     <string>图片显示区</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_openCamera">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>88</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,162,232);
border-radius:4px;
color:white;
</string>
    </property>
    <property name="text">
     <string>Open Camera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_closeCamera">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,162,232);
border-radius:4px;
color:white;</string>
    </property>
    <property name="text">
     <string>Close Camera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_openSeekFace">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,162,232);
border-radius:4px;
color:white;</string>
    </property>
    <property name="text">
     <string>Open SeekFace</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_openVideo">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,162,232);
border-radius:4px;
color:white;</string>
    </property>
    <property name="text">
     <string>Open Video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_chooseImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>88</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,162,232);
border-radius:4px;
color:white;</string>
    </property>
    <property name="text">
     <string>Choose Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Pause">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,162,232);
border-radius:4px;
color:white;</string>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black</string>
    </property>
    <property name="text">
     <string>   基于深度学习的人脸识别 v1.0.0</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSetup">
   <property name="text">
    <string>setup</string>
   </property>
  </action>
  <action name="actionVideo">
   <property name="text">
    <string>Video</string>
   </property>
  </action>
  <action name="actionCamera">
   <property name="text">
    <string>Camera</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
